lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -580,14 +580,19 @@ PushExportResult pushExport(const MObject& mayaObject, const UsdMayaPrimUpdaterC
 
```

Comment:
For the diff in this file, turn off whitespace changes, because a lot of line only changed indenttion.